src/core_plugins/getting_started/public/components/getting_started/getting_started.less

```diff
@@ -0,0 +1,70 @@
+.gettingStartedContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0;
+}
+
+.gettingStartedOptOutContent {
+  position: absolute;
+  top: 20px;
+  right: 30px;
+}
+
+.gettingStartedWelcome {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 200px;
+  width: 100%;
+  background-color: #f7f7f7;
+
+  h1 {
+    transform: translateY(10px);
+  }
+}
+
+.gettingStartedLogo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  border: 1px solid #D7DBDD;
+  width: 100px;
+  height: 100px;
+  border-radius: 100%;
+
+  background-color: #ffffff;
+
+  transform: translateY(-60px);
+}
+
+.gettingStartedLogoRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+
+  img {
+    height: 80px;
+  }
+
+  img + img {
```

Comment:
Same here. We shouldn't style elements directly. I'll add this guidance to our CSS style guide.